docker/jenkins/centos-cuda/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN bash ./install_ccache.sh && rm install_ccache.sh
 # Install Python
 ARG PYTHON_VERSION
 ADD ./install_python.sh install_python.sh
-RUN bash ./install_python.sh && rm install_python.sh
+RUN if [ -n "${PYTHON_VERSION}" ]; then bash ./install_python.sh; fi
+RUN rm install_python.sh
```

Comment:
We'll only really need these if we also make conda builds for centos, or if we start having other centos builds without python. I don't have plans for that right now

Reply:
You're right. Changed everything to keep it in sync.